<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Youtube_dl_Dialog</class>
 <widget class="QDialog" name="Youtube_dl_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>328</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>YOUTUBE-DL UTILITY</string>
  </property>
  <widget class="QGroupBox" name="options_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>159</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Provided Options</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="mp3_radioButton">
      <property name="text">
       <string>mp3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="video_radioButton">
      <property name="text">
       <string>mkv</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="exit_radioButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PROCEED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="youtube_url_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter youtube url :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="youtube_url_liner">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter the save path :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_liner">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="info_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>INFO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_beta">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="system_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>System Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
